Reserve 64999 for documentation (#79)
core-wg/cf-reg-update: draft-ietf-core-cf-reg-update.md

This document updates RFC7252 regarding the registration procedures for the "CoAP Content-Formats" IANA registry, within the "Constrained RESTful Environments (CoRE) Parameters" registry group.
This document also introduces a new column, "Media Type", to the registry.

> Furthermore, this document retires 64999 from the allocatable Content-Format identifiers and reserves its use for documentation.

Reviewer: Can we say this simpler and shorter? If 64999 is reserved for documentation, we don't have to say that it's retired. It's anyway not usable for allocation if it already is reserved for something. E.g.:

```suggestion
Furthermore, this document allocates Content-Format identifier 64999 for use in documentation.
```

Reviewer 2: What was the reason we are not using 65000?

Reviewer: @cabo We use the number in examples of a person requesting a registration within the "FCFS range" of the registry. To make the example convincing, it has to lie within this range. (I.e. in the "old" situation prior to the changes by this document.)
Also the experimental value 65000 might be in actual use in experiments, while 64999 is not supposed to be in use anywhere.

Reviewer 2: Using a code point just for demonstrating how to use a code point causes cognitive dissonance.
So why don't we use 20000?  That also is allocated and FCFS.

In {{iana}}, this document updates {{-coap}} by modifying the registration procedures for the "CoAP Content-Formats" registry to mitigate the risk of unintentional or malicious errors.
These updates amend the different ranges of the registry, introduce a review procedure to be performed for most ranges of the registry, and allow the registration of temporary Content-Format identifiers.
This document also introduces a new column, "Media Type", to the registry.

> Furthermore, this document retires 64999 from the allocatable Content-Format identifiers and reserves its use for documentation.

Reviewer: See prev comment about "retires": seems not needed also here.

> Furthermore, this document retires 64999 from the allocatable Content-Format identifiers and reserves its use for documentation.

Author: ```suggestion
Furthermore, this document reserves Content-Format identifier 64999 for use in documentation.
```

> "Note: As per {{&SELF}}, temporary registrations within the 0-255 range are approved by Designated Experts.
> These registrations are not subject to the formal {{-iana-early}} renewal process."

## Reserving Content-Format Identifier 64999 for Documentation

> IANA is instructed to retire Content-Format identifier 64999 and mark it as reserved for use in documentation.

Reviewer: Same comment as before: better not use "retire" here? We can "allocate" it or "reserve" it.

> IANA is instructed to retire Content-Format identifier 64999 and mark it as reserved for use in documentation.

Author: ```suggestion
IANA is instructed to reserve Content-Format identifier 64999 for use in documentation.
```
